contracts/KlerosPOC.sol

```diff
@@ -167,17 +167,17 @@ contract KlerosPOC is Arbitrator {
         juror.balance+=msg.value;
         pinakion.mint(this,msg.value);
     }
-    
+
     // **************************** //
     // *      Court functions     * //
     // *    Modifying the state   * //
     // **************************** //
-    
+
     /** @dev To call to go to a new period. TRUSTED.
      */
     function passPeriod() public {
-        require(now-lastPeriodChange>=timePerPeriod[uint8(period)]);
-        
+        require(should_pass_period('0x'));
```

Comment:
I wasn't sure how I felt about this. It minimizes code reuse but obfuscates the requirements